packaging/language/npm.py

```diff
@@ -253,7 +253,10 @@ def main():
     elif state == 'latest':
         installed, missing = npm.list()
         outdated = npm.list_outdated()
-        if len(missing) or len(outdated):
+        if len(missing):
+            changed = True
+            npm.install() 
+        elif len(outdated):
             changed = True
```

Comment:
Any chance this PR could be merged?  Issue #1375 is still open, and a related issue is no. 2 result on Google for "ansible npm module", so it's probably happening a lot.